src/TLuaInterpreter.cpp

```diff
@@ -3084,28 +3097,38 @@ int TLuaInterpreter::setLabelCallback(lua_State* L, const QString& funcName)
 
     TEvent event;
     int n = lua_gettop(L);
+    lua_remove(L, 1);
+    lua_remove(L, 1);
```

Comment:
Mind adding a comment here explaining the 'why' behind the two calls?

Reply:
They could probably be combined into one call, but I unfortunately couldn't figure it out. Will be glad to condense if someone knows how! Either way, added a comment!